source/task-crud.js

```diff
@@ -70,59 +81,23 @@ function init() {
                 notes: '',
             };
             
+            // Add the task using the parser and notify listeners
             parser.addTask(task);
             notifyChangeListeners();
 
-            // Disable pop-up after adding task
+            // Hide the add task popup
             addTaskPopup.style.display = "none";
             overlay.style.display = "none";
         }
     });
 
-    // Event listener for confirming the task deletion
```

Comment:
Why were these listeners removed? Now edit and delete no longer works because of that.

Reply:
Jungwoo is right, what was the point of deleting all of these lines?